server/rooms.ts

```diff
@@ -37,7 +37,7 @@ import {RoomGame, RoomGamePlayer} from './room-game';
 import {Roomlogs} from './roomlogs';
 import * as crypto from 'crypto';
 import {RoomAuth} from './user-groups';
-
+import {modlog, ModlogEntry} from './modlog';
```

Comment:
```suggestion
import {modlog, ModlogEntry} from './modlog';

```

There's no need to get rid of the newline @_@